modules/rules_java/7.0.3/presubmit.yml

```diff
@@ -0,0 +1,8 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+tasks:
+  verify_build_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
```

Comment:
```suggestion
    platform: ${{ platform }}
    bazel_version: 6.4.0rc5
```

Reply:
@hvadehra Let's try this.

Reply:
No need to create a new release, we can modify this inline.

Reply:
I lack the rights to make this edit, could you make it?

Reply:
i don't think I do either, I'll just make a new release.

Reply:
It didn't seem to pick it up, does the pipeline read the file from master HEAD?

Reply:
It didn't seem to pick it up, does the pipeline read the file from master HEAD?